confidence/io.py

```diff
@@ -270,7 +270,7 @@ def loads(*strings, missing=Missing.silent):
     return Configuration(*(yaml.safe_load(string) for string in strings), missing=missing)
 
 
-def load_name(*names, load_order=DEFAULT_LOAD_ORDER, extension='yaml', missing=Missing.silent):
+def load_name(*names, load_order=DEFAULT_LOAD_ORDER, extension=('yaml', 'yml'), missing=Missing.silent):
```

Comment:
Maybe this should not be the default. Anyone encountering `yml` files can enable it this way or even hardcode it to just `yml`. The doc states this parameter can be either a `str` or a sequence, should the singular default be a `str` or a one-value `tuple`? :thinking:

Reply:
FXIED, defaulting to just plain old `yaml`, turning this PR into an 'it is now possible to ...' :)